repos/spack_repo/builtin/packages/gaudi/package.py

```diff
@@ -181,10 +182,15 @@ def cmake_args(self):
             self.define_from_variant("GAUDI_USE_DOXYGEN", "docs"),
             # needed to build core services like rndmsvc
             self.define("GAUDI_USE_CLHEP", True),
-            self.define("GAUDI_USE_PYTHON_MAJOR", str(self.spec["python"].version.up_to(1))),
             # todo:
             self.define("GAUDI_USE_INTELAMPLIFIER", False),
         ]
+        # Release notes for v40.0: https://gitlab.cern.ch/gaudi/Gaudi/-/releases/v40r0
+        # Cleanup remaining Python2 support
+        if self.spec.satisfies("@:39"):
+            args.append(
+                self.define("GAUDI_USE_PYTHON_MAJOR", str(self.spec["python"].version.up_to(1)))
+            )
```

Comment:
```suggestion
```
Is it even possible to build with Python 2? Can we just remove this?

Reply:
I checked and all versions in spack have the default value of 3 for this variable.